flight/Modules/Stabilization/stabilization.c

```diff
@@ -420,9 +448,6 @@ static void stabilizationTask(void* parameters)
 						pids[PID_RATE_ROLL + i].iAccumulator = 0;
 					}
 
-					// The unscaled input (-1,1)
```

Comment:
removing this line appears to break horizon mode ;-)

I'm not entirely sure why the compiler doesn't catch there is no variable called raw_input in scope anymore